features/unattached_status.feature

```diff
@@ -28,6 +28,38 @@ Feature: Unattached status
             fips-updates  no         Uncertified security updates to FIPS modules
             livepatch     yes        Canonical Livepatch service
 
+            This machine is not attached to a UA subscription.
+            See https://ubuntu.com/advantage
+            """ 
+    
+    @series.focal
+    Scenario: Unattached status in a focal lxd container
```

Comment:
I wonder too if these tests could be consolidated in a scenario outline too?
What do you think?
```suggestion
    Scenario Outline: Unattached status in focal lxd container

        Given a `focal` lxd container with ubuntu-advantage-tools installed
￼        when I run `ua status` <userspec>
￼        then I will see the following on stdout:
￼            """
￼            SERVICE       AVAILABLE  DESCRIPTION
￼            cc-eal        no         Common Criteria EAL2 Provisioning Packages
￼            esm-apps      yes        UA Apps: Extended Security Maintenance
￼            esm-infra     yes        UA Infra: Extended Security Maintenance
￼            fips          no         NIST-certified FIPS modules
￼            fips-updates  no         Uncertified security updates to FIPS modules
￼            livepatch     yes        Canonical Livepatch service
￼
￼            This machine is not attached to a UA subscription.
￼            See https://ubuntu.com/advantage
￼            """

      Examples:
        | userspec     |
        | as non-root  |
        | with sudo    |
```

Reply:
I think we should not add the Examples here because it would create 2 container for each scenarios in this file, so instead of a total of 2 containers, we'd have 4 containers

Reply:
Good point. +1 on separation unless we determine a set of non-destructive tests on an LXC which allow us to reuse an existing lxc image for multiple scenarios.